app/onboarding/styles.ts

```diff
@@ -1,45 +1,50 @@
 import styled from 'styled-components';
 import COLORS from '@/styles/colors';
 
-export const PageContainer = styled.div`
-  width: 100%;
-  min-height: 100vh;
-  background-color: ${COLORS.seed};
+export const OnboardingContainer = styled.div`
+  min-height: calc(100vh - 60px); // 60px is the hardcoded height of Header
```

Comment:
we should probably change this to use Flex `full-screen` so that there's a single source of truth in terms of the screen height